app/views/posts/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Posts</h1>
-<p>Here is the list for all post of bloger.</p>
+<section id='user-container'>
+<img src="<%= @user.photo %>" alt="image">
+<h1><%= @user.name%></h1>
+</section>
+<p id='user-bio'><%= @user.bio %></>
+<div class='post-container'>
+<% @posts.each_with_index do |post,index| %>
+<li>
+<p>Post #<%= index + 1%></p>
+<h3><%= link_to post.title, controller: "posts", action: "show", user_id: @user.id, post_id: post.id %></h3>
+<p>Comments: <%+ post.comments.length %></p>
```

Comment:
- [ ] When I try to view the Post's **_index_** page, the error below is displayed

![image](https://user-images.githubusercontent.com/77106962/224075997-3352be9a-1d92-42e5-93a9-89fe17015f35.png)

To fix this, kindly change the `post_id` to `id`. The route for Post's _**show**_ page is:

```
 user_post GET  /user/:user_id/posts/:id(.:format) posts#show
```

If you look closely, you would see that there is no param for `post_id` but `id`. Kindly implement this change. 👍 😎